.pipelines/e2e.yaml

```diff
@@ -27,11 +27,16 @@ jobs:
       cd e2e
       go test -timeout 30m -v -run Test_All ./...
     displayName: Run AgentBaker E2E
-  - publish: $(System.DefaultWorkingDirectory)/e2e/vm-logs
-    artifact: vm-logs
-    condition: always()
-  - publish: $(System.DefaultWorkingDirectory)/e2e/cluster-parameters
```

Comment:
was planning on removing cluster param uploads during that other PR, doing it here instead